include/filters.php

```diff
@@ -98,7 +98,7 @@ public function delete_sticky_posts_cache() {
 	 * @since 2.0
 	 *
 	 * @param WP_Comment_Query $query  WP_Comment_Query object.
-	 * @return PLL_Language|false The language to use in the filter, false otherwise.
+	 * @return PLL_Language|string|false The language to use in the filter, false otherwise.
```

Comment:
We should always return an array, either empty, with one or several `PLL_Language` instances.

This would avoid type checking each time we want to use these values, and thus simplify the code. It would also improve PHPStan capability to highlight errors we could write in our code.

Reply:
I'ill review that.